Tasks/AzureRmWebAppDeployment/Utility.ps1

```diff
@@ -2,10 +2,26 @@ $ErrorActionPreference = 'Stop'
 
 function Get-MsDeployExePath
 {
-    $currentDir = (Get-Item -Path ".\").FullName
+    $msDeployExePath = $null
+    try
+    {
+        $msDeployExePath = Get-MSDeployOnTargetMachine
+    }
+    catch [System.Exception]
+    {
+         Write-Verbose ("MSDeploy is not installed in system." + $_.Exception.Message)
+    }
```

Comment:
Can you take a look at the Get-MsDeployLocation in IIS Task msdeployonTargetMachines.ps1 file. You can do it in optimized way.